MagusEncoder/Makefile

```diff
@@ -9,3 +9,6 @@ C_SRC = $(wildcard Src/*.c)
 CPP_SRC = $(wildcard Src/*.cpp)
 
 include $(OPENWARE)/Hardware/f0.mk
+
+upload:
+	$(OPENOCD) -c "program Build/${PROJECT}.elf verify reset exit"
```

Comment:
wrong PR? and is this different from `flash` target?

Reply:
A bonus addition. And that's how I've realized that flashing was in common.mk . Ok, I'll get rid of this, but it wasn't obvious at all - it wasn't present when I've initially used tried building it, before project got upgraded.

Reply:
Dropped that commit